libs/punchapp/src/lib/config/tableConfig.tsx

```diff
@@ -18,58 +37,164 @@ export const tableConfig: GridConfig<Punch> = {
       width: 100,
     },
     {
-      field: 'Priority',
-      valueGetter: (pkg) => pkg.data?.priority1,
+      field: 'Estimate',
+      valueGetter: (pkg) => pkg.data?.estimate,
       width: 100,
     },
     {
-      field: 'PL Sorting',
-      valueGetter: (pkg) => pkg.data?.punchListSorting,
+      field: 'Raised by org',
+      valueGetter: (pkg) => pkg.data?.raisedBy,
+      width: 150,
+    },
+    {
+      field: 'Clearing by org',
+      valueGetter: (pkg) => pkg.data?.cleardBy,
+      width: 150,
+    },
+    {
+      field: 'Cleared',
+      valueGetter: (pkg) => pkg.data?.clearedAtDate,
+      cellRenderer: (props: ICellRendererProps<Punch, string | null | undefined>) => {
+        return props.value ? new Date(props.value).toLocaleDateString : '';
+      },
       width: 100,
     },
     {
-      field: 'PL Type',
-      valueGetter: (pkg) => pkg.data?.punchListType,
+      field: 'Verified',
+      valueGetter: (pkg) => pkg.data?.verifiedAtDate,
+      cellRenderer: (props: ICellRendererProps<Punch, string | null | undefined>) => {
+        return props.value ? new Date(props.value).toLocaleDateString : '';
+      },
       width: 100,
     },
     {
-      field: 'Estimate',
-      valueGetter: (pkg) => pkg.data?.estimate,
+      field: 'Handover plan',
+      valueGetter: (pkg) => pkg.data?.handoverPlan,
+      cellRenderer: (props: ICellRendererProps<Punch, string | null | undefined>) => {
+        return props.value ? new Date(props.value).toLocaleDateString : '';
+      },
+      width: 110,
+    },
+    {
+      field: 'Form type',
+      valueGetter: (pkg) => pkg.data?.formularType,
+      valueFormatter: (pkg) =>
+        pkg.data?.commissioningPackageId
+          ? proCoSysUrls.getCommPkgUrl(pkg.data.commissioningPackageId)
+          : '',
+      cellRenderer: (props: ICellRendererProps<Punch, string | null | undefined>) => {
+        if (props.valueFormatted && props.value) {
+          return <LinkCell url={props.valueFormatted} urlText={props.value} />;
+        }
+        return null;
+      },
       width: 100,
     },
     {
-      field: 'Raised by org',
-      valueGetter: (pkg) => pkg.data?.raisedByOrganization,
+      field: 'Tag',
+      valueGetter: (pkg) => pkg.data?.tagNo,
+      valueFormatter: (pkg) =>
+        pkg.data?.commissioningPackageId
+          ? proCoSysUrls.getCommPkgUrl(pkg.data.commissioningPackageId)
+          : '',
+      cellRenderer: (props: ICellRendererProps<Punch, string | null | undefined>) => {
+        if (props.valueFormatted && props.value) {
+          return <LinkCell url={props.valueFormatted} urlText={props.value} />;
+        }
+        return null;
+      },
       width: 150,
     },
     {
-      field: 'Clearing by org',
-      valueGetter: (pkg) => pkg.data?.clearingByOrganization,
+      field: 'Commpkg',
+      valueGetter: (pkg) => pkg.data?.commissioningPackageNo,
+      valueFormatter: (pkg) =>
+        pkg.data?.commissioningPackageId

```

Comment:
use commissioningPackageUrlId instead. We used commissioningPackageId before, but its value has been changed to be a random GUID, and not the actual procosys ID that is used for the procosys URL. That's why we introduced commissioningPackageUrlId.